openslides/core/views.py

```diff
@@ -353,3 +359,19 @@ def check_permissions(self, request):
         if (self.action in ('create', 'update', 'destroy') and
                 not request.user.has_perm('core.can_manage_tags')):
             self.permission_denied(request)
+
+
+class UrlPatternsView(utils_views.APIView):
+    """
+    Returns a dictonary with all url patterns as json.
+    """
+    URL_KWARGS_REGEX = re.compile(r'%\((\w*)\)s')
+
+    def get_context_data(self, **context):
+        result = {}
+        url_dict = get_resolver(None).reverse_dict
+        for url_pattern in filter(lambda key: isinstance(key, str), url_dict.keys()):
```

Comment:
Are you sure?
```
>>> from django.core.urlresolvers import get_resolver
>>> url_dict = get_resolver(None).reverse_dict
>>> list(filter(lambda key: isinstance(key, str), url_dict.keys()))[10]
'projector_clean_scroll'
```
This shows that the result of iterating the filter function are some strings. In my example this string with underscores is exactly the same as [this URL pattern name](https://github.com/OpenSlides/OpenSlides/blob/master/openslides/projector/urls.py#L51). The URL with slashes would be `/projector/clean/scroll/. The URL pattern would be the result of [this function call](https://github.com/OpenSlides/OpenSlides/blob/master/openslides/projector/urls.py#L48). You can get access to the URL and the URL pattern (and its kwargs and this stuff) by evaluating the `url_dict` with help of the URL pattern name as key (or even the view function because all URL patterns are registered twice by Django).